### PR TITLE
Remove extra whitespace from text inputs in Buffer share

### LINKED ISSUE #5830: Remove extra spaces in text input areas on /internal/articles
**Is your feature request related to a problem? Please describe.**
As a DEV staff member using `/internal/articles`, I am annoyed by the additional characters present in the "Share to Buffer" text input areas.

**Describe the solution you'd like**
Extra spacing characters are removed in the "pre-filled" text.  Screenshots below.

Additionally, it would be helpful if these text input areas were a bit wider by default.

**Additional context**
- Very similar to https://github.com/thepracticaldev/dev.to/issues/4276
- Instructions on how to reach this page ([are available here](https://github.com/thepracticaldev/dev.to/issues/4276#issuecomment-539179806)) 

### Current
Twitter Main and Satellite current
Facebook current

### Desired
Twitter Main and Satellite desired
Facebook desired

### PR DESCRIPTION
**Pull request by citizen428**
## What type of PR is this? (check all applicable)

- [X] Bug Fix

## Description

When adding articles to Buffer via `/internal/articles` there was some extra whitespace at the end. This PR takes care of this.

## Related Tickets & Documents

Closes #5830

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

**Before**

Screen Shot 2020-02-06 at 12 21 31

Screen Shot 2020-02-06 at 12 07 09

**After**

Screen Shot 2020-02-06 at 13 32 53
Screen Shot 2020-02-06 at 13 22 23

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [ ] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Comment by citizen428** [comment]
Sorry, there was a lot of re-formatting going on when I saved, let me know if you want me to to undo that. I'll leave comments on the things that are actually important for the purpose of this PR.

File: app/views/internal/articles/_individual_article.html.erb [hunk 1]
```diff
@@ -63,158 +66,178 @@
...
-              <textarea cols="37" rows="6" wrap="hard" name="tweet" maxlength="255"><%= article.title %>
-                <% if article.user.twitter_username? %>&#013{ author: @<%= article.user.twitter_username %> } #DEVCommunity<% end %>
+      <div>
+        <%= form_with url: "/internal/buffer_updates", html: { data: { action: "submit->article#highlightElement" } } do %>
+        <input type="hidden" name="social_channel" value="main_twitter" />
+        <input type="hidden" name="article_id" value="<%= article.id %>" />
+        <h5>Twitter MAIN</h5>
+        <div class="form-group">
+          <textarea cols="37" rows="6" wrap="hard" name="tweet" maxlength="255">
+                <% concat article.title
```
Comment (citizen428) [inline app/views/internal/articles/_individual_article.html.erb]: A `textarea` tag preserves indentation between different ERb tags. The [`concat`](https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-concat) helper allows us to build up a string inside a single tag.

I could have added a helper function for this purpose, but I'm not a big fan of their global namespace, and the two places where we need this are slightly different.

File: app/views/internal/articles/_individual_article.html.erb [hunk 2]
```diff
@@ -63,158 +66,178 @@
...
-            <textarea cols="37" rows="6" wrap="hard" name="fb_post" required>
-            </textarea>
-          </div>
-          <button class="btn btn-primary mb-2">📘 Post to Facebook</button>
+        <input type="hidden" name="social_channel" value="satellite_twitter" />
+        <input type="hidden" name="article_id" value="<%= article.id %>" />
+        <h5>Twitter Satellite</h5>
+        <div class="form-group">
+          <textarea cols="37" rows="6" wrap="hard" name="tweet" maxlength="255">
+                  <% concat article.title
```
Comment (citizen428) [inline app/views/internal/articles/_individual_article.html.erb]: This is the same `concat` trick as above.

File: app/views/internal/articles/_individual_article.html.erb [hunk 3]
```diff
@@ -63,158 +66,178 @@
...
+        <% end %>
         <% end %>
+      </div>
+
+      <%= form_with url: "/internal/buffer_updates", html: { data: { action: "submit->article#highlightElement" } } do %>
+      <input type="hidden" name="social_channel" value="facebook" />
+      <input type="hidden" name="article_id" value="<%= article.id %>" />
+      <h5>Facebook & LinkedIn</h5>
+      <div class="form-group">
+        <textarea cols="37" rows="6" wrap="hard" name="fb_post" required></textarea>
```
Comment (citizen428) [inline app/views/internal/articles/_individual_article.html.erb]: Putting the closing tag right after the opening tag removed the extra spaces.

**Comment by citizen428** [comment]
@rhymes That's for @pkfrank to decide, but at least there are no extra lines anymore. I'll check tomorrow if pulling up the closing `</textarea>` tag removes it.

**Comment by jacobherrington** [comment]
I agree we should check out @rhymes' comment before merging, but it is an improvement either way. :smile:

**Pull request merged**